### DISCUSSION TITLE
Unnecessary individuals for Issuer role

### DISCUSSION THREAD

**Discussion by rivettp** [edmcouncil/fibo#1390] [Financial Business and Commerce (FBC)] [answered]
There is nothing in FIBO that requires a role to have a separate individual as opposed to a class. If it's something specific to each "Contract" then that would require a separate Issuer individual for each Share - and the proposal has only one AlphabetIncIssuer reused for both Class A and Class C Alphabet Shares, which is not consistent with the justification. If the individual can be reused in this scenario for multiple contracts the individual for the business Entity can also be reused. If more sophisticated scenarios need an individual per contract then that can be represented as appropriate.

_Originally posted by @rivettp in https://github.com/edmcouncil/fibo/pull/864_

**Comment by trypuz** [comment]
@rivettp, what exactly should be done to fix this issue?

**Comment by ElisaKendall** [comment, marked as answer]
This issue is more complicated than @rivettp has said.  A party to a contract is a role that they are playing, and the same party can be a party to many contracts, but that is still a role, not an independent business entity.  Thus, the same issuer can issue multiple classes of stock, whose primary distinction has to do with voting rights in this case.  But, the business entity isn't a role, it is independent.  Thus at least one issuer / contract party individual is required in addition to the individual for the legal entity.  This is straightforward, following the standard party / role model used for the last 30 years in information modeling.

I believe this should be closed, no change.  @MikeHypercube - what do you think?
